src/coreclr/nativeaot/docs/interop.md

```diff
@@ -4,28 +4,28 @@
 
 The PInvoke calls in AOT compiled binaries are bound lazily at runtime by default for better compatibility. The AOT compiler
 can be configured to generate direct calls for selected PInvoke methods that are bound during startup. The unmanaged libraries
-and entrypoints referenced via direct calls have to be always available at runtime, otherwise the native binary fails to start.
+and entry points referenced via direct calls always have to be available at runtime, otherwise the native binary fails to start.
```

Comment:
@dotnet/interop-contrib Is there a preferred spelling for `entry*point` in the docs?

The official docs have a mix of `entrypoint`, `entry point` and `entry-point`. For example, the recent https://github.com/dotnet/docs/pull/30549 prefers `entry-point`.

Reply:
I'm no expert in languages, but at least [Wikipedia](https://en.wikipedia.org/wiki/Entry_point) and [Oxford](https://oxfordlearnersdictionaries.com/definition/english/entry-point) seem to prefer `entry point`, whereas Merriam-Webster doesn't have a listing for the noun at all.

Reply:
I have been doing `entry-point` is the adjective and `entry point` is the noun. The change to `entry points` in this case would match that.